hammer_cli_katello.gemspec

```diff
@@ -65,6 +65,8 @@ Gem::Specification.new do |gem|
   gem.version = HammerCLIKatello.version
   gem.required_ruby_version = '>= 2.7', '< 3.2'
 
+  gem.add_dependency 'theforeman-rubocop', '~> 0.1.0'
```

Comment:
This should have been a development dependency, not a runtime dependency

Reply:
https://github.com/Katello/hammer-cli-katello/pull/950